src/nano/rpc.py

```diff
@@ -3425,8 +3427,6 @@ def send(self, wallet, source, destination, amount, work=None, id=None):
                     (wallet, source, destination, and amount) is undefined
                     behavior and may result in an error in the future.
 
-        :type send_id: str
```

Comment:
This should be renamed to `:type id: str` as it had the wrong name.

Reply:
added back